doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -981,106 +1003,98 @@ notations are given below. The optional :n:`@scope` is described in
+
 At any time, the interpretation of a notation for a term is done within
-a *stack* of interpretation scopes and lonely notations. In case a
-notation has several interpretations, the actual interpretation is the
-one defined by (or in) the more recently declared (or opened) lonely
-notation (or interpretation scope) which defines this notation.
-Typically if a given notation is defined in some scope ``scope`` but has
-also an interpretation not assigned to a scope, then, if ``scope`` is open
-before the lonely interpretation is declared, then the lonely
-interpretation is used (and this is the case even if the
-interpretation of the notation in scope is given after the lonely
-interpretation: otherwise said, only the order of lonely
-interpretations and opening of scopes matters, and not the declaration
-of interpretations within a scope).
+a *stack* of notation scopes and lonely notations. If a
+notation is defined in multiple scopes, |Coq| uses the interpretation from
+the most recently opened notation scope or declared lonely notation.
+
+Use the :cmd:`Print Visibility` command to display the current notation scope stack.
 
 .. cmd:: Open Scope @scope
 
-   The command to add a scope to the interpretation scope stack is
-   :n:`Open Scope @scope`.
+   Adds a scope to the notation scope stack.
+
+   When the command appears in a section, the scope is only added within the section.
+   When the command is outside a section, the scope is exported as part of the current module.
+
+   This command supports the :attr:`local` and :attr:`global` attributes.
+   :attr:`local` adds the scope only in the current module.  :attr:`global`
+   adds the scope outside all currently open sections in the innermost open
+   module.  :attr:`global` has no effect if the command is not inside a section.
 
 .. cmd:: Close Scope @scope
 
-   It is also possible to remove a scope from the interpretation scope
-   stack by using the command :n:`Close Scope @scope`.
+   Removes a scope from the notation scope stack.
 
    Notice that this command does not only cancel the last :n:`Open Scope @scope`
    but all its invocations.
 
-.. note:: ``Open Scope`` and ``Close Scope`` do not survive the end of sections
-          where they occur. When defined outside of a section, they are exported
-          to the modules that import the module where they occur.
+   .. todo PR: invocations?
 
-.. cmd:: Local Open Scope @scope.
-         Local Close Scope @scope.
+   When the command appears in a section, the removal applies only within the section.
 
-   These variants are not exported to the modules that import the module where
-   they occur, even if outside a section.
+   .. todo PR I don't understand how closing a scope relates to exporting a module
 
-.. cmd:: Global Open Scope @scope.
-         Global Close Scope @scope.
+   .. todo PR When the command is outside a section, the removal
+      is exported as part of the current module.  What does that mean?
```

Comment:
I've already answered these two `todo` but since you added `PR`, my answers do not show anymore in the Changes tab.

Reply:
It's still present here: https://github.com/coq/coq/pull/11718#discussion_r413667817 but hard to find since it's marked "outdated".  I knew roughly where it was so it didn't take too long to find.  It's not present in the files view.

Oddly, I don't recall reading the comment earlier even though I go through the comments sequentially.

Perhaps the lesson is to avoid changing these anchor lines unnecessarily.  When practical.

Reply:
I think I got it, please take another look.